### ISSUE TITLE
KeyError: link while processing a feed

### ISSUE THREAD

**Issue by fluffy-critter** [PlaidWeb/Pushl#5]
```
Traceback (most recent call last):
  File "/home/fluffy/.local/share/virtualenvs/pushl-jjqXtYO_/lib/python3.6/site-packages/feedparser.py", line 398, in __getattr__
    return self.__getitem__(key)
  File "/home/fluffy/.local/share/virtualenvs/pushl-jjqXtYO_/lib/python3.6/site-packages/feedparser.py", line 356, in __getitem__
    return dict.__getitem__(self, key)
KeyError: 'link'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/fluffy/.local/share/virtualenvs/pushl-jjqXtYO_/bin/pushl", line 11, in <module>
    sys.exit(main())
  File "/home/fluffy/.local/share/virtualenvs/pushl-jjqXtYO_/lib/python3.6/site-packages/pushl/__main__.py", line 123, in main
    worker.wait_finished()
  File "/home/fluffy/.local/share/virtualenvs/pushl-jjqXtYO_/lib/python3.6/site-packages/pushl/__main__.py", line 60, in wait_finished
    queued.result()
  File "/usr/lib/python3.6/concurrent/futures/_base.py", line 425, in result
    return self.__get_result()
  File "/usr/lib/python3.6/concurrent/futures/_base.py", line 384, in __get_result
    raise self._exception
  File "/usr/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/fluffy/.local/share/virtualenvs/pushl-jjqXtYO_/lib/python3.6/site-packages/pushl/__main__.py", line 82, in process_feed
    self.submit(self.process_entry, entry.link)
  File "/home/fluffy/.local/share/virtualenvs/pushl-jjqXtYO_/lib/python3.6/site-packages/feedparser.py", line 400, in __getattr__
    raise AttributeError("object has no attribute '%s'" % key)
AttributeError: object has no attribute 'link'
```

**Comment by fluffy-critter** [comment]
Fixed in ff2a28e3120df68e5446c3d633526ebc30d1ef6c

**Comment by fluffy-critter** [comment]
well no it was actually fixed in a refactor a long time ago, the bug I just fixed was something else oops
